_infra/helm/frontstage/values.yaml

```diff
@@ -96,4 +96,7 @@ gnu:
   fingerprint: ED1B7A3EADF95687
 
 password:
-  attemptTimeout: 86400
\ No newline at end of file
+  attemptTimeout: 86400
+
+frontstageProfiler:
+  enabled: true
```

Comment:
This should default to our production configuration of false. It will then be overridden as required.